src/journal/compress.c

```diff
@@ -112,7 +112,7 @@ int compress_blob_lz4(const void *src, uint64_t src_size,
         if (src_size < 9)
                 return -ENOBUFS;
 
-        r = LZ4_compress_limitedOutput(src, dst + 8, src_size, (int) dst_alloc_size - 8);
+        r = LZ4_compress_limitedOutput(src, (char*)dst + 8, src_size, (int) dst_alloc_size - 8);
```

Comment:
so far we used "char" only for actual characters, and "uint8_t" as generic byte-sized type referencing "arbitrary bytes"...